### PR TITLE
Introduce probe url request for proxied external requests to confirm url...

### PR DESCRIPTION
**Pull request by anguss00**
... is good before downloading

Fixes issue #1624

### CODE REVIEW

File: test/unit/au/org/emii/portal/proxying/RequestProxyingControllerTests.groovy [hunk 1]
```diff
@@ -9,8 +9,12 @@ package au.org.emii.portal.proxying
 
 import grails.test.*
 
+import static au.org.emii.portal.HttpUtils.Status.HTTP_500_INTERNAL_SERVER_ERROR
```
Comment (danfruehauf) [inline test/unit/au/org/emii/portal/proxying/RequestProxyingControllerTests.groovy]: You may include `au.org.emii.portal.HttpUtils.Status.*` here. Wouldn't make much of a difference and less likely to break.

File: test/unit/au/org/emii/portal/proxying/RequestProxyingControllerTests.groovy [hunk 2]
```diff
@@ -9,8 +9,12 @@ package au.org.emii.portal.proxying
...
+
 class RequestProxyingControllerTests extends GrailsUnitTestCase {
+
     void testPerformProxyingAddsFileDownloadCookie() {
+
```
Comment (danfruehauf) [inline test/unit/au/org/emii/portal/proxying/RequestProxyingControllerTests.groovy]: Redundant new line.

File: test/unit/au/org/emii/portal/proxying/RequestProxyingControllerTests.groovy (hunk 1 above)
Comment (anguss00) [inline test/unit/au/org/emii/portal/proxying/RequestProxyingControllerTests.groovy]: Sure thing, good point

**Comment by danfruehauf** [comment]
@anguss00 Please show me what's the best way to test it when you have time.

**Comment by anguss00** [comment]
No worries, I'll make those changes now and show you in 5 min

**Comment by anguss00** [comment]
Changes made @danfruehauf

**Pull request merged**